rosetta/services/tx_operation.go

```diff
@@ -382,8 +455,9 @@ var (
 	// where the sender is a contract address. This is also known as ops for an 'internal' transaction.
 	// All operations have at least 7 elements on the stack when executed.
 	internalNativeTransferEvmOps = map[string]interface{}{
-		vm.CALL.String():     struct{}{},
-		vm.CALLCODE.String(): struct{}{},
+		vm.CALL.String():         struct{}{},
```

Comment:
'Create' and 'Create2' opcode are missing.
'Callcode' does not really transfer Ether.